### ISSUE TITLE
Reyhan last of Abazan moving counters that don't exist.

### ISSUE THREAD

**Issue by Thedrummonger** [magefree/mage#4055]
Whenever a token creture dies on my side of the field Reyhan last of Abzans ability (Whenever a creature you control dies or is put into the command zone, if it had one or more +1/+1 counters on it, you may put that many +1/+1 counters on target creature.) triggers even when the token does not have any counters on it. After I select a creature to move the counters to, it places two counters on the creature that seemingly came from no were. 
Here is a video of the bug in action https://www.youtube.com/watch?v=qAzRbFWmnHw

**Comment by theelk801** [comment]
Has this happened before or since? I can't seem to recreate it.

**Comment by Thedrummonger** [comment]
1.4.26V4 was when I first noticed it, but I assumed it was just an my imagination until it happened again in 1.4.26V5. I did testing in that version and then 1.4.26V6 since it came out during testing. I was able to reproduce in both versions. Mycoloth was used to create the tokens in both of my tests, I'm not sure if it happens with tokens from other sources. The deck is centered around +1/+1 counter so there are a lot of creatures on the field with +1/+1 counters but in all my test I have made sure the token that dies has no counters (as seen in the video). Here is the deck I'm using if that helps any http://tappedout.net/mtg-decks/partner-commander-abzan/
